src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -929,6 +941,45 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
             @Config.LangKey("config." + MOD_ID + ".client.display.rottenFoodOverlayColor")
             public int rottenFoodOverlayColor = 0x88CC33;
         }
+    
+        public static final class GrassColorCFG
+        {
+        	@Config.Comment("If true, grass and foliage will be slightly varied in color.")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.noiseEnable")
+         	public boolean noiseEnable = true;
+	    	
+	    	@Config.Comment("If true, grass and foliage will be colored seasonally.")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.seasonColorEnable")
+         	public boolean seasonColorEnable = true;
+	    	
+	    	@Config.Comment("The noise scale. Default = 10")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.noiseScale")
+         	public float noiseScale = 10f;
+	    	
+	    	@Config.Comment("How many darkness levels should the noise have? Default = 5")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.noiseLevels")
+         	public int noiseLevels = 5;
+	    	
+	    	@Config.Comment("How potent should the darkness be? Default = 0.15")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.noiseDarkness")
+         	public float noiseDarkness = 0.15f;
+	    	
+	    	@Config.Comment("Color data for summer coloring. Syntax: RED,BLUE,GREEN,ALPHA. Default=30,255,30,28")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.seasonColorSummer")
+         	public String seasonColorSummer = "30,255,30,28";
+	    	
+	    	@Config.Comment("Color data for summer coloring. Syntax: RED,BLUE,GREEN,ALPHA. Default=255,168,30,128")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.seasonColorAutumn")
+         	public String seasonColorAutumn = "255,168,30,128";
+	    	
+	    	@Config.Comment("Color data for summer coloring. Syntax: RED,BLUE,GREEN,ALPHA. Default=33,165,255,48")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.seasonColorWinter")
+         	public String seasonColorWinter = "33,165,255,48";
+	    	
+	    	@Config.Comment("Color data for summer coloring. Syntax: RED,BLUE,GREEN,ALPHA. Default=0,255,160,64")
+         	@Config.LangKey("config." + MOD_ID + ".client.grassColor.seasonColorSpring")
+         	public String seasonColorSpring = "0,255,160,64";
```

Comment:
Instead of string, which requires parsing and error handling, I recommend using hexadecimal here, example:

```java
@Config.Comment("Color data for summer coloring. Syntax: ALPHA, RED, GREEN, BLUE.")
@Config.LangKey("config." + MOD_ID + ".client.grassColor.seasonColorSummer")
public int seasonColorSummer = 0x1C1E1EFF
```

This also follows ARGB standard, which iirc have some libs in java.